Jenkinsfile

```diff
@@ -157,6 +157,10 @@ node('vets-website-linting') {
   }
 
   stage('Deploy') {
+    if (isContentTeamUpdate()) {
+      throw new hudson.AbortException("content branches fail intentionally")
+    }
+
```

Comment:
So this would appear to prevent a wip or content branch from getting deployed, but I don't see how that will prevent failing tests from being merged? Eventually someone is going to merge this content branch into master and then it will make it to prod again.